modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -7,9 +7,8 @@
 
 The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
 
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
-The default value, `cluster`, is supported for both parameters.
-This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
+* OVN-Kubernetes does not support setting the internal traffic policy for a Kubernetes service to `local`.
+This limitation can affect you when you add a service of type `ClusterIP`, `LoadBalancer`, `NodePort`, or add a service with an external IP.
```

Comment:
I'd reword this to avoid "you," since the process is being impacted, not the user. A minor nit.